Refit/SystemTextJsonContentSerializer.cs

```diff
@@ -44,10 +44,13 @@ public Task<HttpContent> SerializeAsync<T>(T item)
 
             JsonSerializer.Serialize(utf8JsonWriter, item, jsonSerializerOptions);
 
-            var content = new StreamContent(utf8BufferWriter.DetachStream())
+            var stream = utf8BufferWriter.DetachStream();
+
+            var content = new StreamContent(stream)
             {
                 Headers =
                 {
+                    ContentLength = stream.Length,
```

Comment:
Is `Length` always available here?

Reply:
Yep, comes from the `PooledMemoryStream`:

https://github.com/reactiveui/refit/blob/c2e2c92cc0c9971bb3922d773c7bda7b9259fbd8/Refit/Buffers/PooledBufferWriter.Stream.cs#L60

Which is returned by the call to `DetachStream()`:

https://github.com/reactiveui/refit/blob/c2e2c92cc0c9971bb3922d773c7bda7b9259fbd8/Refit/Buffers/PooledBufferWriter.cs#L104-L110